collector/info_schema_innodb_metrics.go

```diff
@@ -0,0 +1,65 @@
+// Scrape `information_schema.innodb_metrics`
+
+package collector
+
+import (
+	"database/sql"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	infoSchemaInnodbMetricsQuery = `
+		SELECT
+		  name, subsystem, type, comment,
+		  count
+		  FROM information_schema.innodb_metrics
+		  WHERE status = 'enabled'
+		`
+)
+
+func ScrapeInnodbMetrics(db *sql.DB, ch chan<- prometheus.Metric) error {
```

Comment:
Generally, it would be nice for all exported functions/methods/types to have doc comments.

`golint` currently says:

```
collector/binlog.go:33:1: exported function ScrapeBinlogSize should have comment or be unexported
collector/engine_tokudb.go:37:1: exported function ScrapeEngineTokudbStatus should have comment or be unexported
collector/global_status.go:61:1: exported function ScrapeGlobalStatus should have comment or be unexported
collector/global_variables.go:19:1: exported function ScrapeGlobalVariables should have comment or be unexported
collector/info_schema_auto_increment.go:40:1: exported function ScrapeAutoIncrementColumns should have comment or be unexported
collector/info_schema_innodb_metrics.go:21:1: exported function ScrapeInnodbMetrics should have comment or be unexported
collector/info_schema_processlist.go:153:1: exported function ScrapeProcesslist should have comment or be unexported
collector/info_schema_query_response_time.go:37:1: exported function ScrapeQueryResponseTime should have comment or be unexported
collector/info_schema_tables.go:64:1: exported function ScrapeTableSchema should have comment or be unexported
collector/info_schema_tablestat.go:41:1: exported function ScrapeTableStat should have comment or be unexported
collector/info_schema_userstats.go:109:1: exported function ScrapeUserStat should have comment or be unexported
collector/slave_status.go:22:1: exported function ScrapeSlaveStatus should have comment or be unexported
```